invenio_users_resources/ext.py

```diff
@@ -52,3 +59,71 @@ def init_resources(self, app):
             service=self.groups_service,
             config=GroupsResourceConfig,
         )
+
+    def init_db_hooks(self, app):
+        """Initialize the database hooks for reindexing updated users/roles."""
+        # make sure that the hooks are only registered once per DB connection
+        # and not once per app
```

Comment:
i think this wouldn't actually hurt, but it would cause every user/role to be reindexed multiple times (once per active invenio flask app; depending on the setup that'd be UI and API)